src/main/java/no/fdk/utils/GraphUtils.java

```diff
@@ -11,6 +11,9 @@
 
 public class GraphUtils {
 
+    private GraphUtils() {
+    }
```

Comment:
lombok's @NoArgsConstructor?

Reply:
Just noticed it was private default. In that case you can:
`@NoArgsConstructor(extraPrivate = true)`

Reply:
Nice one!